deeppavlov/configs/ranking/rel_ranking.json

```diff
@@ -1,88 +0,0 @@
-{
```

Comment:
Please ask Dmitry Evseev about this file

Reply:
@dmitrijeuseew